src/cmakeProject.ts

```diff
@@ -513,6 +686,17 @@ export class CMakeProject {
     }
     private readonly _ctestEnabled = new Property<boolean>(false);
 
+    /**
```

Comment:
What are these for?

Reply:
I don't know either. For package presets, I am only following the same as for test presets.
For test presets, it looks like these boolean variables are set depending on the presence on disk of some cmake files. CPack seems to consume similar input files as CTest and I kept the same coding. But even for test, I was not able to figure out what are they used for. Only what are they set to. I think we'll end up removing both.